### PR TITLE
Add Nhost auth example to the playground

### PR DESCRIPTION
**Pull request by nunopato**
- Adds an example with Nhost Auth to the playground

### CODE REVIEW

File: web/src/components/Nhost/Nhost.js
Comment (Tobbe) [inline web/src/components/Nhost/Nhost.js:26]:
```suggestion
```

File: web/src/components/Nhost/Nhost.js
Comment (Tobbe) [inline web/src/components/Nhost/Nhost.js:116]: Did you want `skipFetchCurrentUser` here?

Comment (nunopato) [reply web/src/components/Nhost/Nhost.js:116]: This example only uses Nhost auth so not needed, I think.

**Review by Tobbe — APPROVED** [review]

**Comment by Tobbe** [comment]
This is looking good! I've approved the PR, but let's wait with merging until the implementation has landed in a RW release.

**Comment by thedavidprice** [comment]
@Tobbe the Nhost PR has been merged into main. How would you like to handle the timing for merging this?

**Comment by Tobbe** [comment]
@thedavidprice I'd like to wait until a proper release of RW has been made, so we can upgrade the version of RW used in the playground to something that supports NHost before merging

**Comment by morganmspencer** [comment]
@Tobbe did you want me to look into getting this integrated with the latest updates?

**Comment by Tobbe** [comment]
> @Tobbe did you want me to look into getting this integrated with the latest updates?

Yes please, that'd be awesome

**Comment by morganmspencer** [comment]
@Tobbe this might be a noob question on my part but how do I push the changes to this PR? I've got them all set locally but it's blocking me from pushing

**Comment by Tobbe** [comment]
> @Tobbe this might be a noob question on my part but how do I push the changes to this PR? I've got them all set locally but it's blocking me from pushing

You probably can't :(  Only project maintainers can usually do that. Easiest is to just do a new, separate, PR. Mention it here when it's up, so we get cross-links. Thanks!

**Pull request closed without merging**
